intro-to-rails-6/intro-to-rails-api.md

```diff
@@ -94,14 +126,14 @@ So lets write a test to specify what we **want** to be rendered back to us on `g
 
 ### So... What Do We Want
 
-With your neighbor answer the following questions:
+Write down the answer the following questions:
```

Comment:
If we want them to write something we should probably create a space for them to write stuff out.

```suggestion
Think about the answers to the following questions:
```

Reply:
I'd just intended this to be notes they're taking.